name-badges/makebadges.py

```diff
@@ -0,0 +1,35 @@
+#!/bin/pyhton3
+
+import sys
+import re
+import lxml.etree as ET
+import subprocess
+#from cairosvg import svg2png
+
+namesfile = sys.argv[1]
+
+names=open(namesfile).readlines()
+
+def altersvg(file,name,role):
+    svg = open(file, mode="rb").read()
+    root = ET.fromstring(svg)
+    print(root)
+    root.findall(".//text[@inkscape:label='Name']",root.nsmap)[0][0].text=name
+    root.findall(".//text[@inkscape:label='Role']",root.nsmap)[0][0].text=role
+    return(ET.tostring(root, encoding='UTF-8'))
+
+
+
+subprocess.call(["mkdir", "tmp"])
```

Comment:
`os.mkdir` is a function https://docs.python.org/3/library/os.html#os.mkdir

Reply:
It looks like you might enjoy using the [`tempfile`](https://docs.python.org/3/library/tempfile.html) module though -- that avoids doing any directory munging yourself, as well as ensuring cleanup happens even if things error.